src/Extensions/Hosting.Services/HostBuilderExtensions.cs

```diff
@@ -25,57 +26,57 @@ public static class HostBuilderExtensions
 		/// <summary>
 		/// Add actions that will be executed inside service fabric service RunAsync method
 		/// </summary>
-		public static ServiceFabricHostBuilder<TContext> AddServiceAction<TContext>(
-			this ServiceFabricHostBuilder<TContext> builder,
-			Func<TContext, CancellationToken, Task> action)
-				where TContext : ServiceContext =>
-					builder.AddServiceAction(new ServiceAction<TContext>(action));
+		/// <typeparam name="TService">Type of service fabric service, currently should be only StatelessService or StatefulService</typeparam>
```

Comment:
>StatelessService [](start = 90, length = 16)

`<see cref="StatelessService"/>` will add hyperlinks in the documentation

Reply:
Updated, thanks